src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { getTodos } from './api';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [qwery, setQwery] = useState('');
+  const [selectedFilter, setSelectedFilter] = useState('all');
+  const [checkLoad, setCheckLoad] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```
I recommend you read about naming variables in the right way. [Here](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx) you can find a small guide on naming variables.